frontend/packages/dev-console/src/components/dropdown/ResourceDropdown.tsx

```diff
@@ -77,7 +77,11 @@ class ResourceDropdown extends React.Component<ResourceDropdownProps, State> {
 
     // If autoSelect is true only then have an item pre-selected based on selectedKey.
     if (autoSelect) {
-      const selectedItemKey = !selectedKey ? _.get(_.keys(this.state.items), 0) : selectedKey;
+      const dropdownItem =
+        this.props.loaded && _.isEmpty(this.state.items)
+          ? this.props.actionItem.actionKey
```

Comment:
please add conditional check for `this.props.actionItem` === undefined/null

Reply:
The `actionKey` and `actionTitle` has to be sent with `actionItem`, it's mandatory. So if we check for `actionItem` that should be enough

Reply:
This should not be this.props.loaded because it can be false initially so the loaded from the new props should be checked. And if you see we already have a check for loaded above so we dont need to add this check here.